### PR TITLE
darkmode button in mobile view

### PR DESCRIPTION
**Pull request by HeinKhantZaw**
Fix for Issue #1214

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -106,7 +106,7 @@
         <div class="buttons">
           <a
             class="toggle-theme"
-            href="#toggleTheme"
+            href="#"
```
Comment (shahednasser) [inline index.html:109]: Can you change it back? Now when clicking on the button it takes you to the top of the page

Comment (HeinKhantZaw) [reply index.html:109]: ok. I'll add it back.

**Comment by HeinKhantZaw** [comment]
btw @shahednasser , Have you checked the website in your mobile phone? There's a horizontal scroll bar when the website is opened in mobile browser. Shall I create a new issue for that too? Although it is not related with my current pull request, I think I should let you know about this responsive issue in mobile devices.

IMG_20201223_195459

**Comment by shahednasser** [comment]
Hello @HeinKhantZaw sorry for the delay. As for the horizontal scrolling bug, I will create a new issue with it and if you'd like you can comment on it to take it up.

**Comment by shahednasser** [comment]
Great work thank you!

@all-contributors please add @HeinKhantZaw for code

**Pull request merged**